kokoro/builds/upload_symbols.sh

```diff
@@ -72,6 +72,7 @@ function upload_symbol_file {
 function upload_debug_symbols {
   local api_key=$1
   local bin_folder=$2
```

Comment:
We also can change the second argument to the `build_folder`, and add '/bin' or '/lib' suffix below (when call upload_symbol_file function). But I don't have a strong opinion on this, so whatever you prefer.

Reply:
I thought about this. The main reason why I did it that way, was to break if I missed a usage of that function.